include/xgboost/context.h

```diff
@@ -74,6 +109,12 @@ struct DeviceOrd {
         return DeviceSym::CPU();
       case DeviceOrd::kCUDA:
         return DeviceSym::CUDA() + (':' + std::to_string(ordinal));
+      case DeviceOrd::kSyclDefault:
```

Comment:
What is "sync:{ordinal}"?

Reply:
It means using the SYCL device with index `{ordinal}`. It can be CPU or GPU, depending on user's system settings.